search.html

```diff
@@ -0,0 +1,12 @@
+<script src="constable.js"></script>
+<script src="https://code.jquery.com/jquery-2.2.2.min.js"
+integrity="sha256-36cp2Co+/62rEAAYHLmRCPIych47CvdM+uTBJwSzWjI="
+crossorigin="anonymous"></script>
+
+<script>
+
+</script>
+
+<input type='text' id='search'></input>
```

Comment:
Minor: be consistent with your quotes. We use double quotes in html but whichever you choose, just be consistent.